app/src/main/java/ch/hikemate/app/ui/map/MapScreen.kt

```diff
@@ -246,17 +266,23 @@ fun MapScreen(
       // Maximum number of bytes that can be used by the tile file system cache. Default is 600MB.
       tileFileSystemCacheMaxBytes = 600L * 1024L * 1024L
     }
+
+    if (authViewModel.currentUser.value == null) {
```

Comment:
Same question as in the `HikeDetailsScreen`

Reply:
Answered above (will resolve it)